scripts/bluetooth

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*
+import subprocess
+from bluetool import Bluetooth
+bluetooth = Bluetooth()
+
+fs_s=str(bluetooth.get_connected_devices())
+no_el=int(len(bluetooth.get_connected_devices()))
+fs_len=len(str(bluetooth.get_connected_devices()))
+
+place=fs_s.find('name') 
+#print(place)
+#print(fs_s[place+8])
+
+res = [i for i in range(len(fs_s)) if fs_s.startswith("name", i)]
+#print(res)
+l_r=len(res)
+d_name=""
+for x in range(0, l_r):
+	#print(fs_s[res[x]+8])
+	start=res[x]+8
+	end=res[x]+8
+	for y in range(start,fs_len):
+		if fs_s[y]=="\'":
+			end=y
+			break
+	for z in range(start,end):
+		d_name+=fs_s[z]
+	#d_name+=" "
+
+
+cmd='bluetooth | grep -wo \'on\' | tr \'\\n\' \'\\0\' || true'
+result = subprocess.check_output(cmd, shell=True)
+#print(result)
+#print(len(d_name))
+len_dname=len(d_name)
+if result=='on\0':
+	if len_dname!=0:
+		print("  "+d_name)
+
+cmd_button="if [ ! -z \"$button\" ]; then /usr/bin/i3-msg -q exec /usr/bin/gnome-control-center bluetooth; fi"
```

Comment:
Can we maybe do this natively in Python instead of shelling out? Also, please make sure the blocklet is only reacting to a left-click and nothing else. We have an outstanding PR for adjusting this for other blocklets too :slightly_smiling_face:

Reply:
I actually really like the ability to open up the settings by scrolling. I'll try to make it work so that it works by left-clicking only.
I could not found a way to emulate the button in python natively.

Reply:
You basically just have to read the environment variable and then send a command to i3 using IPC. We're using [i3pc](https://pypi.org/project/i3ipc/) in [another blocklet](https://github.com/regolith-linux/regolith-i3xrocks-config/blob/d6555032e2de3889eb0eff89b621b4946e807694/scripts/focused-window-name) for this already

Reply:
Thanks for the reference. I'll look into the example and implement it in the script.